ietf/doc/tests_ballot.py

```diff
@@ -360,7 +360,7 @@ def test_request_last_call(self):
         self.assertTrue('aread@' in outbox[-1]['Cc'])
 
     def test_edit_ballot_writeup(self):
-        draft = IndividualDraftFactory(states=[('draft','active'),('draft-iesg','iesg-eva')])
+        draft = IndividualDraftFactory(states=[('draft','active'),('draft-iesg','iesg-eva')], stream=StreamName.objects.get(slug='ietf'))
```

Comment:
I think using `stream_id='ietf'` should work here and would avoid the `StreamName` import and db lookup.